portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,10 @@ function addRandomFunFact() {
   const funfactContainer = document.getElementById('funfact-container');
   funfactContainer.innerText = funfact;
 }
+
+function getRandomQuoteUsingArrowFunctions() {
```

Comment:
I would just call the function getRandomQuotes. The fact that you use lambdas is an implementation details of the function and does not impact the caller of this function.

Reply:
done